test_staticjinja.py

```diff
@@ -131,9 +133,12 @@ def watch(self):
 def test_should_handle(reloader, template_path):
     template1_path = str(template_path.join("template1.html"))
     test4_path = str(template_path.join("test4.html"))
+
+    open(test4_path, 'w').close()
```

Comment:
I'm honestly not sure that this is a good idea, but it's the best way I could think of to get the tests passing after I changed the logic to check for the file's existence. Thoughts?